doc/user_guide/docker/docker-usage.md

```diff
@@ -0,0 +1,146 @@
+# AI-Lab Docker Edition
+
+Using Exasol AI-Lab Docker Edition requires some specific [prerequisites](prerequisites.md) but also offers additional benefits.
+
+The [Operating System and Setup Guide](os-setup.md) helps you with the initial system setup.
+
+AI-Lab also offers a [short introduction](intro.md) to Docker Images and Containers if you are new to this technology.
+
+## Defining Environment Variables
+
+The Unix shell commands in the following sections will use some environment variables. By this you can adapt the commands to your specific preferences while still being able to execute them literally:
+* Variable `VERSION` refers to the version of Exasol AI-Lab Docker Edition you want to use, alternatively you can use `latest`.
+* Variable `VOLUME` is expected to contain the name of your Docker volume, see [Managing User Data](managing-user-data.md).
+* Variable `LISTEN_IP` defines the range of IP-addresses allowed to connect to the forwarded Jupyter port.
+  * `0.0.0.0` means all IP-addresses are allowed.
+  * For local setups, we recommend `127.0.0.1`.
+  * Please contact your IT department if there are security restrictions.
+
+Here is an example:
+
+```shell
+VERSION=0.2.0
+LISTEN_IP=0.0.0.0
+VOLUME=my-vol
+```
+
+## Creating a Docker Container from the Docker Image
+
+You can use an Exasol database with AI-Lab in two ways
+* [External Exasol database](#ai-lab-with-external-exasol-database)
+* Integrated [Exasol Docker-DB](#ai-lab-managing-exasol-docker-db-internally) managed by AI-Lab
+
+### AI-Lab with External Exasol Database
+
+In this scenario the AI-Lab Docker container does not need access to the Docker daemon.
+
+The following command will
+* Download the Docker image for the specified version `$VERSION` of the AI-Lab if the image of the specified version is not yet available in your Docker service
+* Run a Docker container using this image
+* Mount the volume `$VOLUME` to the directory `/root/notebooks` inside the container
+  * If the volume does not exist yet, then it will be created automatically.
+* Forward port `49494` on the [daemon machine](prerequisites.md) to allow connections from all IP addresses matched by `$LISTEN_IP`
+
+```shell
+docker run \
+  --volume ${VOLUME}:/root/notebooks \
+  --publish ${LISTEN_IP}:49494:49494 \
+  exasol/ai-lab:${VERSION}
+```
+
+If you want to use a newer version of the AI-Lab then please [delete Docker volumes](managing-user-data.md#replacing-the-docker-volume) created with older versions.
```

Comment:
```suggestion
If you want to use a newer version of the AI-Lab then please [delete the Docker volumes](managing-user-data.md#replacing-the-docker-volume) created with older versions.
```

However, this really the correct behavior, because they might lose all their progress.